pipeline/requirements.txt

```diff
@@ -5,26 +5,17 @@ bioutils==0.1.4
 bx-python==0.7.3
 certifi==2016.8.2
 configparser==3.5.0
-CrossMap>=0.2.4
+CrossMap==0.2.6
 Cython==0.24.1
-Django==1.10
-django-cors-headers==1.1.0
-django-jwt-auth==0.0.2
-django-utils==0.0.2
-djangorestframework==3.4.1
-djangorestframework-jwt==1.8.0
 docutils==0.12
-hgvs==0.4.9
-leiden==1.0.0
+hgvs==0.3
 lockfile==0.12.2
 luigi==2.2.0
 nose==1.3.7
 numpy==1.11.1
 Parsley==1.3
-pip==8.1.2
-psycopg2==2.6.2
+pip==9.0.1
 pygr==0.8.2
-pyhgvs==0.9.4
```

Comment:
this seems odd -- i know pyhgvs is used during brca_pseudonym_generator and some other places throughout the pipeline

Reply:
We might be able to get rid of this dependency in the future, now that Reese Hart's hgvs library converts to hg38 coordinates.  The reason why we were using pyhgvs (the Counsyl library) before was that the hgvs library didn't support hg38 yet.

Reply:
0.9.4 doesn't seem to be in PyPi, therefore it is built from source instead: https://github.com/BRCAChallenge/brca-exchange/pull/617/files#diff-8f09b8970829c955182f646b3026bccbR42

Good to know that we can get rid of it, I found it quite confusing to have hgvs and pyhgvs in the same code ;)